certificate-rotation.md

```diff
@@ -16,24 +16,20 @@ This topic describes how to rotate Application Service Adapter for VMware Tanzu
     ```yaml
     api_ingress:
     tls:
-        crt: |
-          NEW-API-TLS-CRT
-        key: |
-          NEW-API-TLS-KEY
+        secret_name: NEW-API-TLS-SECRET-NAME
+        namespace: NEW-API-TLS-SECRET-NAMESPACE
     app_ingress:
     tls:
-        crt: |
-          NEW-APP-TLS-CRT
-        key: |
-          NEW-APP-TLS-KEY
+        secret_name: NEW-APP-TLS-SECRET-NAME
+        namespace: NEW-APP-TLS-SECRET-NAMESPACE
     ```
 
     Where:
 
-    - `NEW-API-TLS-CRT` is the PEM-encoded public certificate for the Application Service Adapter API.
-    - `NEW-API-TLS-KEY` is the PEM-encoded private key for the Application Service Adapter API.
-    - `NEW-APP-TLS-CRT` is the PEM-encoded public certificate for applications deployed using the Application Service Adapter.
-    - `NEW-APP-TLS-KEY` is the PEM-encoded private key for applications deployed using the Application Service Adapter.
+    - `NEW-API-TLS-SECRET-NAME` is the secret containing the PEM-encoded public certificate for the Application Service Adapter API.
```

Comment:
For this parameter and for the app-ingress one, we should specify that the secret must be of [type `kubernetes.io/tls`](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets) so that the user knows the correct data fields for the secret to contain.

Reply:
Good idea. Fixed